content/en/service_management/workflows/actions/set_variables.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Set variables
+disable_toc: false
+algolia:
+  tags: ['workflow variables', 'variables', 'mutable']
+further_reading:
+- link: "/service_management/workflows/actions/flow_control#for-loop"
+  tag: "Documentation"
+  text: "Use a for loop to perform an action iteratively"
+
+---
+
+To set a mutable workflow variable, use the [Set variable][1] action. You can use the action to declare, update and access variables throughout your workflow. Setting a variable allows you to perform more complex workflow operations. For example:
+- _Handling API pagination_: API requests sometimes require you to keep track of a page token or offset.
+- _Handling lists_: You can use a variable to initialize an array and perform actions like map and reduce.
+- _Iteration_: Variables allow you to manipulate and store data inside a [for loop][2] and then use that data in the rest of the workflow.
+
+## Set a variable
+
+To set a variable:
+1. Click the plus (**+**) icon on your workflow canvas to open the action catalog.
+1. Search for and select the **Set variable** step.
+1. Click the **Set variable** step and enter a **Step name**.
+1. Enter a **variable name**. Variable names must be strings with alphanumeric characters and underscores and must start with a letter.
+1. Enter a value for the variable.
+   - Type ``{{`` if you want to use a workflow context variable.
+   - To create an object, click the **Create object** <i class="icon-api"></i> button.
+   - To create an array, click the **Create array** <span id="icon-array">[ ]</span> button.
+
+If you need to change the value of a variable after setting it, you must add an additional **Set variable** step and either reassign the variable or create a new variable. The example workflow below:
+- Sets a variable called `int_list` with a list of integers: `[1,2,3,4]`
```

Comment:
I think you should rename `int_list` to `intList` because you are using `intList` in the enums below.